operators/cmd/instance-operator/main.go

```diff
@@ -85,6 +86,7 @@ func main() {
 	flag.StringVar(&containerEnvVncImg, "container-env-vnc-img", "crownlabs/tigervnc", "The image name for the vnc image (sidecar for graphical container environment)")
 	flag.StringVar(&containerEnvWebsockifyImg, "container-env-websockify-img", "crownlabs/websockify", "The image name for the websockify image (sidecar for graphical container environment)")
 	flag.StringVar(&containerEnvNovncImg, "container-env-novnc-img", "crownlabs/novnc", "The image name for the novnc image (sidecar for graphical container environment)")
+	flag.StringVar(&containerEnvFileMgrImg, "container-env-filemanager-img", "filebrowser/filebrowser", "The image name for the filebrowser image (sidecar for graphical container environment)")
```

Comment:
As you are introducing new parameters, those should also be added to the instance-operator helm chart, in order to allow for their configuration.

Some pointers:
* Helm global values: https://github.com/netgroup-polito/CrownLabs/blob/master/deploy/crownlabs/values.yaml
* Helm values: https://github.com/netgroup-polito/CrownLabs/blob/master/operators/deploy/instance-operator/values.yaml
* Helm deploy template: https://github.com/netgroup-polito/CrownLabs/blob/master/operators/deploy/instance-operator/templates/deployment.yaml

Reply:
Is it ok to have v2.15.0 as the default value in the Helm values file?

Reply:
I would leave the default to latest, and override it in the actual inventory in CrownOps. Otherwise this will get soon stale